getting-started/modules/core/outbox.md

```diff
@@ -0,0 +1,32 @@
+# Outbox
+
+## Dependencies
+
+Valtimo contains an implementation of
+the [transactional outbox pattern](https://microservices.io/patterns/data/transactional-outbox.html). To make use of
```

Comment:
I don't think we should be linking to an external site for vital information like this, but rather have our own page. I'm fine with this as a temp solution though.

@marijnritense any thoughts?